stories/components/wizard/docs/examples/custom_stepper.tsx

```diff
@@ -45,7 +45,7 @@ const Demo = () => {
         <Wizard.Sidebar title='Site Audit Settings'>
           <Wizard.Stepper step={1} onActive={handleStepChange(1)}>
             Personal
-            <Text color='text-secondary-invert' fontWeight={400} tag='div'>
+            <Text color='text-secondary-invert' textAlign='left' fontWeight={400} tag='div'>
```

Comment:
@ilyabrower Ilia, can we add textAlign='left' style by default to Text inside Stepper?

I tried adding it in wizard styles for SText descendants inside SStepper. But it didn't work, so I add it in the example only 🤔 But I think it's not a good solution since we can set this by default.